### PR TITLE
New chat fix

### PR DESCRIPTION
**Pull request by OmarBasem**
Fixes app crash when pressing new chat and bottom sheet not showing properly

### QA review notes:
1. Currently the bottom button is not positioned properly (overlapping with safe area) as we are currently using an old bottom sheet.
2. There is a 1 second delay when pressing "New chat". That's because it is trying to open a bottom sheet from inside a bottom sheet. We would have to migrate it to a screen component. Opened an issue https://github.com/status-im/status-mobile/issues/14741

### CODE REVIEW

**Comment by cammellos** [comment]
@OmarBasem make sure once approved QA takes a look at this one before merging, thanks for the fix!

File: src/status_im/ui/screens/home/sheet/views.cljs
Comment (J-Son89) [inline src/status_im/ui/screens/home/sheet/views.cljs:76]: can we create an issue to properly address this? This seems like a temporary fix to me.

**Review by J-Son89 — APPROVED** [review]

File: src/status_im/ui/screens/home/sheet/views.cljs
Comment (OmarBasem) [reply src/status_im/ui/screens/home/sheet/views.cljs:76]: Yes it is temporary. WIll create an issue.

File: src/status_im2/subs/contact.cljs
Comment (ibrkhalil) [inline src/status_im2/subs/contact.cljs]: Please rename this sub, As I am adding the deleted sub of `:sorted-and-grouped-by-first-letter` that you accidentally deleted again [here](https://github.com/status-im/status-mobile/pull/14680)

**Review by ibrkhalil — CHANGES_REQUESTED** [review]

**Comment by qoqobolo** [comment]
Hi @OmarBasem, thanks for the fix!

Q: There are several issues related to the `Start new chat` flow itself. For example, incorrect work of checkboxes on the `New chat` bottom sheet or the ability to start a chat with a user who hasn't yet approved your contact request.
Should I report such issues here, or is that out of the scope of this PR?

**Comment by ibrkhalil** [comment]
> Hi @OmarBasem, thanks for the fix!
> 
> Q: There are several issues related to the `Start new chat` flow itself. For example, incorrect work of checkboxes on the `New chat` bottom sheet or the ability to start a chat with a user who hasn't yet approved your contact request. Should I report such issues here, or is that out of the scope of this PR?

Most of them are fixed [here](https://github.com/status-im/status-mobile/pull/14680)

**Comment by qoqobolo** [comment]
> Most of them are fixed https://github.com/status-im/status-mobile/pull/14680

Ah, cool! Then it would make sense now to merge the fix only for the crash (after addressing your comment above) and fix all the remaining bugs (if any) separately.

**Comment by ibrkhalil** [comment]
> > Most of them are fixed #14680
> 
> Ah, cool! Then it would make sense now to merge the fix only for the crash (after addressing your comment above) and fix all the remaining bugs (if any) separately.

Yes, But this fix is going to break my PR, That's why I suggested a new subscription name .. Because the one I created uses this same name

**Comment by qoqobolo** [comment]
@ibrkhalil yeah, let's wait for @OmarBasem's answer. 
After changing the subscription name I'll re-check the PR again briefly.

File: src/status_im2/subs/contact.cljs
Comment (OmarBasem) [reply src/status_im2/subs/contact.cljs]: Fixed

**Comment by OmarBasem** [comment]
> Hi @OmarBasem, thanks for the fix!
> 
> Q: There are several issues related to the `Start new chat` flow itself. For example, incorrect work of checkboxes on the `New chat` bottom sheet or the ability to start a chat with a user who hasn't yet approved your contact request. Should I report such issues here, or is that out of the scope of this PR?

As @ibrkhalil has mentioned, they should be fixed in his PR

**Comment by OmarBasem** [comment]
Hey @qoqobolo, please let me know if all is okay now.

FYI, this PR fixes the crash that is happening when clicking on "new chat", and makes the bottom sheet properly visible (previously is was squashed to the bottom)

**Comment by qoqobolo** [comment]
> Hey @qoqobolo, please let me know if all is okay now.

Yep, LGTM, thank you! PR can be merged.

**Review by ibrkhalil — APPROVED** [review]

**Comment by OmarBasem** [comment]
@qoqobolo thank you!

**Pull request merged**
